### PR TITLE
cleanup, atlas edit requests, duplicate entries, etc

### PR DESCRIPTION
**Pull request by GitFincy**
removed twmqkp, txg90x, 000358, twobkt and twmq4p, as these are all duplicate entries in two seperate art pieces. requested by `Eragon#7668` in thread https://discord.com/channels/960791635342524496/960847771106152488/962249177004384256

### CODE REVIEW

**Comment by Fincent7** [comment]
updated the description of tx8baa as requested by `Wiggler Enthusiast#1985` in thread https://discord.com/channels/960791635342524496/960847771106152488/962324338978201600

**Comment by Fincent7** [comment]
Removed a duplicate Chibidoki entry
Removed Haruka Karibu duplicate entries
Merged two F1 Raceway entries
Removed a duplicate r/trees entry
Merged two Slime Stack entries
Removed multiple entries of the Colorado University Logo
Removed a Linux place entry
Removed a Loky Ronin entry
Removed an Octoling Octopus entry
Removed & Merged Dynamix entries (& fixed some linebreaks)
Removed & Merged Mana Nagase entries
Merged two 2balkan4you entries
Removed & Merged Chloe entries

File: web/atlas.json [hunk 1]
```diff
@@ -5231,7 +5213,6 @@
...
 {"id": "txbnj7", "submitted_by": "Wrath_FMA", "name": "CatTail|Plants vs. Zombies", "description": "CatTail, The evolved form of Lily pad from Plants vs. Zombies", "website": "", "subreddit": "/r/PlantsVSZombies", "center": [1465.5, 987.5], "path": [[1452.5, 975.5], [1452.5, 998.5], [1478.5, 998.5], [1478.5, 975.5], [1452.5, 975.5]]},
-{"id": "txbn0e", "submitted_by": "ProfessionalFish8505", "name": "Octoling Octopus", "description": "Pixel art based on Octolings from Splatoon 2. Specifically, this is the octopus form they can turn into to swim through ink or superjump.", "website": "", "subreddit": "/r/splatoon", "center": [1127.5, 1856.5], "path": [[1119.5, 1849.5], [1119.5, 1863.5], [1135.5, 1863.5], [1135.5, 1849.5], [1119.5, 1849.5], [1119.5, 1849.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:5234]: This has info that the other description doesn't. Merge the two descriptions to preserve that info.

File: web/atlas.json [hunk 2]
```diff
@@ -5616,7 +5596,6 @@
...
 {"id": "txeepb", "submitted_by": "ININHAHALELE", "name": "Moona Hoshinova", "description": "Moona Hoshinova (1st Gen.) is a talent from the Idol VTuber group, Hololive. She is the first Hololive Talent from Indonesia to reach a milestone of a million subscribers.", "website": "https://hololive.hololivepro.com/en/talents?gp=indonesia", "subreddit": "/r/hololive", "center": [1384.5, 995.5], "path": [[1381.5, 990.5], [1386.5, 990.5], [1389.5, 998.5], [1379.5, 998.5], [1380.5, 992.5], [1383.5, 994.5], [1384.5, 993.5], [1388.5, 996.5], [1388.5, 997.5], [1387.5, 996.5], [1382.5, 996.5]]},
-{"id": "txeej9", "submitted_by": "AsukaMainGG", "name": "Mana Nagase", "description": "Mana Nagase was an idol hailing from the Japanese idol franchise IDOLY PRIDE. She was voiced by the late Sayaka Kanda.\n \nShe currently has four songs (First Step, Precious, Memories of the Starry Sky, and Song For You) and two covers (Racing into the Night by YOASOBI and Ghost in a Flower by Yorushika)", "website": "", "subreddit": "/r/IdolyPride", "center": [1497.5, 1242.5], "path": [[1489.5, 1234.5], [1489.5, 1249.5], [1505.5, 1249.5], [1505.5, 1234.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:5619]: This has info that the other description doesn't. Merge the two descriptions to preserve that info.

Comment (AnonymousRandomPerson) [inline web/atlas.json:5619]: This has info that the other description doesn't. Merge the two descriptions to preserve that info.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Resolve comments.

File: web/atlas.json (hunk 1 above)
Comment (GitFincy) [reply web/atlas.json]: Got it, thank you for pointing it out. Resolved in my last commit: https://github.com/placeAtlas/atlas/pull/1146/commits/5ba9a5ebf4d7ca121e77159b0eaa82a6e4b0981e

File: web/atlas.json (hunk 2 above)
Comment (GitFincy) [reply web/atlas.json]: Got it, thank you again for pointing it out. Resolved in my last commit: https://github.com/placeAtlas/atlas/pull/1146/commits/5ba9a5ebf4d7ca121e77159b0eaa82a6e4b0981e

**Comment by GitFincy** [comment]
Resolved comments & Resolved merge conflict.

**Pull request merged**